docs-app/.template-lintrc.js

```diff
@@ -2,4 +2,13 @@
 
 module.exports = {
   extends: 'recommended',
+
+  overrides: [
+    {
```

Comment:
Can you link the related issue here? (about `.gts`)

Reply:
Posted over on https://github.com/ember-template-lint/ember-template-lint/pull/2844#issuecomment-1505300332